boards/nxp/kw45b41z_evk/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright 2024 NXP
```

Comment:
this needs splitting up into multiple commits, one for adding soc, one for adding board, one for making test changes

Reply:
Sure.. will do that